pubnub-memberships/src/test/kotlin/com/pubnub/entities/MembershipExtensionKtTest.kt

```diff
@@ -0,0 +1,211 @@
+package com.pubnub.entities
+
+import com.pubnub.api.PNConfiguration
+import com.pubnub.api.PubNub
+import com.pubnub.api.endpoints.objects.member.GetChannelMembers
+import com.pubnub.api.endpoints.objects.member.ManageChannelMembers
+import com.pubnub.api.endpoints.objects.membership.GetMemberships
+import com.pubnub.api.endpoints.objects.membership.ManageMemberships
+import com.pubnub.api.endpoints.remoteaction.ExtendedRemoteAction
+import com.pubnub.api.models.consumer.objects.channel.PNChannelMetadata
+import com.pubnub.api.models.consumer.objects.member.PNMember
+import com.pubnub.api.models.consumer.objects.member.PNMemberArrayResult
+import com.pubnub.api.models.consumer.objects.membership.PNChannelMembership
+import com.pubnub.api.models.consumer.objects.membership.PNChannelMembershipArrayResult
+import com.pubnub.api.models.consumer.objects.uuid.PNUUIDMetadata
+import com.pubnub.entities.models.consumer.membership.FetchMembershipsResult
+import com.pubnub.entities.models.consumer.membership.MembershipsResult
+import com.pubnub.entities.models.consumer.membership.SpaceIdWithCustom
+import com.pubnub.entities.models.consumer.membership.UserIdWithCustom
+import io.mockk.MockKAnnotations
+import io.mockk.every
+import io.mockk.impl.annotations.MockK
+import io.mockk.spyk
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+
+internal class MembershipExtensionKtTest {
```

Comment:
Initially I did it but I changed it because having current name allows switching (using Ctrl+Shift+t in Intellij) between MembershipExtensionKtTest and MembershipExtension.